src/main/resources/META-INF/persistence.xml

```diff
@@ -17,48 +17,49 @@
   ~ SPDX-License-Identifier: Apache-2.0
   ~ Copyright (c) OWASP Foundation. All Rights Reserved.
   -->
-<persistence xmlns="http://xmlns.jcp.org/xml/ns/persistence"
+<persistence xmlns="https://jakarta.ee/xml/ns/persistence"
```

Comment:
`xmlns` should remain at `http://xmlns.jcp.org/xml/ns/persistence`. Essentially, DataNucleus (which is using this file) has not yet moved to Jakarta itself. So this should remain at its pre-Jakarta value. I ran into this too, during the migration work in v4.12. :sweat_smile:

Reply:
Whoops, duplicate comment.